discovery-provider/src/utils/ipfs_lib.py

```diff
@@ -79,50 +79,21 @@ def get_metadata(self, multihash, default_metadata_fields, user_replica_set=None
         retrieved_from_gateway = False
         retrieved_from_ipfs_node = False
         start_time = time.time()
+        retrieved_metadata = False
 
-        with concurrent.futures.ThreadPoolExecutor(max_workers=2) as executor:
-            metadata_futures = {}
-            metadata_futures[
-                executor.submit(
-                    self.get_metadata_from_ipfs_node, multihash, default_metadata_fields
```

Comment:
let's remove that connect peer from index_network_peers?

https://github.com/AudiusProject/audius-protocol/blob/f1abd8ba272f9ce43564f9cda946ac272ec700d4/discovery-provider/src/tasks/index_network_peers.py#L120

Reply:
discussed, punting for now

Reply:
discussed and we'll punt this since it doesn't affect indexing perf rn